components/pedigree/resources/src/main/resources/pedigree/editor.css

```diff
@@ -1176,3 +1201,16 @@
   min-width: 200px;
   max-width: 550px;
 }
+.at-age-input, .cause-input {
+    margin-top: 3px;
+    width: 120px !important;
+    height: 15px !important;
```

Comment:
This makes the placeholders unreadable (at least for me). 
![image](https://cloud.githubusercontent.com/assets/651980/21691595/3badf546-d347-11e6-90ae-3a06c5cde978.png)
Remove  `height: 15px !important;`. You may want to add `padding: 0` to override the `.3em` default if the inputs seem too tall.